### PR TITLE
add .env

### PR DESCRIPTION
**Pull request by donguks**
Add REACT_APP_JSON_RPC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ This repository contains ElYFI web interface source code.
 ELYFI is adding real estate to the DEFI concept. This is the expansion of the current crypto-to-crypto applications as ELYFI will introduce traditional assets to the open financial market.
 
 ## Setup
+- touch .env file in the project directory and add the following environment variables:
```
Comment (bigint) [inline README.md:10]:
```suggestion
- Create `.env` file in the project directory and add the following environment variables:
```

touch sounds very technical and some users might not understand it 🙇🏻

Comment (donguks) [reply README.md:10]: Thanks!

**Comment by JUNG-gyucheol** [comment]
l

**Comment by JUNG-gyucheol** [comment]
ㅣ

**Review by JUNG-gyucheol — APPROVED** [review]

**Pull request merged**